pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -217,7 +212,7 @@ sub create_purchase_link {
 sub _suggest_imunify360 {
     my ($self) = @_;
 
-    my $is_kernelcare_needed = _needs_kernelcare();
+    my $is_kernelcare_needed = Whostmgr::Imunify360->new()->needs_kernelcare();
```

Comment:
Although I'm sure something else is surely loading this module, it may not be a bad idea to `use Whostmgr::Imunify360` in this module in case any upstream deps which pull this in change in a manner which makes that assumption no longer true.

Reply:
It gets loaded through `Cpanel::LoadModule::load_perl_module`.  The entire module is a noop if it fails to load.

Reply:
Gotcha, didn't spot that earlier. Consider this change to have my blessing :)